hu.js

```diff
@@ -1,215 +1,232 @@
 /* http://Github.com/Canop/hu.js */
 
 // A simple SVG library by denys.seguret@gmail.com
-;(function(){
-
-	"use strict";
-	
-	var nn = 1, // counter for dynamically generated def id
-		U = function(n){ this.n = n },
-		fn = U.prototype,
-		nopx = { // css properties which don't need a unit
+(() => {
```

Comment:
The semicolon at start ensured there was no misparsing when the file was concatenated to another one